src/test/java/seedu/contax/testutil/Assert.java

```diff
@@ -29,6 +29,7 @@ public static void assertThrows(Class<? extends Throwable> expectedType, Executa
     public static void assertThrows(Class<? extends Throwable> expectedType, String expectedMessage,
             Executable executable) {
         Throwable thrownException = Assertions.assertThrows(expectedType, executable);
+        System.out.println(thrownException.getMessage());
```

Comment:
Is this `println()` intentional or for logging purposes?

Reply:
yes because I'm currently trying to figure out why this doesn't build on the windows CI, will remove it once I fix it